src/routes/version_creation.rs

```diff
@@ -20,7 +20,7 @@ use sqlx::postgres::PgPool;
 use validator::Validate;
 
 lazy_static! {
-    static ref RE_URL_SAFE: Regex = Regex::new(r"^[a-zA-Z0-9_\-.]*$").unwrap();
+    static ref RE_URL_SAFE: Regex = Regex::new(r"\S").unwrap();
```

Comment:
Again, this is not URL SAFE;
This is the second time I see the same regex in a lazy static, would it be better to move this to a utility class with all of the more common regexes?